src/admin/js/ets-admin.js

```diff
@@ -0,0 +1,37 @@
+jQuery(document).ready(function ($) {
+    //On click on refresh button then ajax call
+    jQuery('#wl-refresh-summary').on('click', function(e) {
+        e.preventDefault();
+        let postId = jQuery(this).data('post-id');
```

Comment:
@ravisoni6262 is this file being transpiled? `let` may still pose some issues on [older browsers](https://caniuse.com/#feat=let).